services/bots.py

```diff
@@ -85,6 +98,15 @@ def get_repo_appropriate_bot_token(
     repo: Repository,
     installation_name_to_use: Optional[str] = GITHUB_APP_INSTALLATION_DEFAULT_NAME,
 ) -> Tuple[Dict, Optional[Owner]]:
+
+    log.info(
+        "Get repo appropriate bot token",
```

Comment:
Are both this token + gh app installation safe things to log?

Reply:
I believe so, I think the token isn't generated til here:
https://github.com/codecov/worker/blob/70645fde638d559494b93f4ee9693847d52446ad/services/bots.py#L165-L170

where installation_name_to_use I think is a GH specific string:
https://github.com/codecov/worker/blob/305c1eb744e3cf0cf3165528a8a5ea83fdb5603a/services/notification/notifiers/base.py#L53